README.md

```diff
@@ -1 +1,24 @@
-# rails-blog
\ No newline at end of file
+# README
+
```

Comment:
- Please use the [Microverse template](https://github.com/microverseinc/readme-template) to update your README file for a professional repo.
_Kindly keep in mind that this is could invalidate your PR, but since it's the last day of the week and to prevent a repetition!